### ISSUE TITLE
[BUG] incompatible protobuf dep version when installing cudf

### ISSUE THREAD

**Issue by pxLi** [rapidsai/cudf#10990]
**Describe the bug**
To use conda install cudf=22.08/22.06 will introduce incompatible protobuf version as dependency,
```
conda install -c conda-forge mamba
mamba install -y -c rapidsai -c rapidsai-nightly -c nvidia -c conda-forge -c defaults cudf=22.08 python=3.8 cudatoolkit=11.2
```

which gave us protobuf==3.19.4

when we try `import cudf`, we will get,
```
>>> import cudf
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/conda/lib/python3.8/site-packages/cudf/__init__.py", line 71, in <module>
    from cudf.io import (
  File "/opt/conda/lib/python3.8/site-packages/cudf/io/__init__.py", line 8, in <module>
    from cudf.io.orc import read_orc, read_orc_metadata, to_orc
  File "/opt/conda/lib/python3.8/site-packages/cudf/io/orc.py", line 14, in <module>
    from cudf.utils.metadata import (  # type: ignore
  File "/opt/conda/lib/python3.8/site-packages/cudf/utils/metadata/orc_column_statistics_pb2.py", line 7, in <module>
    from google.protobuf.internal import builder as _builder
```

![image](https://user-images.githubusercontent.com/8086184/170614466-1ea00de4-c5a0-4bdd-af1d-660e64434f5a.png)

and this also happens to 22.06 intermittently, if we are lucky enough we could get protobuf>=3.20+, but not always

manual run `pip install --upgrade protobuf` to upgrade protobuf version (3.20+) could resolve this issue. 
Seems this is like one issue before https://github.com/rapidsai/cudf/issues/8570

**Steps/Code to reproduce bug**
```
conda install -c conda-forge mamba
mamba install -y -c rapidsai -c rapidsai-nightly -c nvidia -c conda-forge -c defaults cudf=22.08 python=3.8 cudatoolkit=11.2

python
import cudf
```

**Expected behavior**
import cudf correctly

**Environment overview (please complete the following information)**
nvidia/cuda:11.2.2-runtime-centos7

**Comment by shwina** [comment]
Hi @pxLi -- is this installing into a fresh environment or an existing one? Perhaps you could try `mamba create` rather than `mamba install`?

**Comment by pxLi** [comment]
> Hi @pxLi -- is this installing into a fresh environment or an existing one? Perhaps you could try `mamba create` rather than `mamba install`?

hi, I guess this does not help consistently. ref: https://github.com/rapidsai/cudf/issues/8570#issuecomment-878473067
Pin the correct version in cudf conda env should help in this case

**Comment by galipremsagar** [comment]
resolved in #10996
